openapi/tests/openapi_integration/test_payload_operations.py

```diff
@@ -13,266 +15,315 @@ def setup():
     drop_collection(collection_name=collection_name)
 
 
+def assert_points(points, with_vectors=False):
+    ids = [point["id"] for point in points]
+
+    if not with_vectors:
+        for point in points:
+            point["vector"] = None
+
+    response = request_with_validation(
+        api="/collections/{collection_name}/points",
+        method="POST",
+        path_params={"collection_name": collection_name},
+        body={"ids": ids, "with_vector": False, "with_payload": True},
+    )
+    assert response.ok
+
+    assert sorted(response.json()["result"], key=itemgetter("id")) == sorted(
+        points, key=itemgetter("id")
+    )
+
+
 def test_payload_operations():
     # create payload
     response = request_with_validation(
-        api='/collections/{collection_name}/points/payload',
+        api="/collections/{collection_name}/points/payload",
         method="POST",
-        path_params={'collection_name': collection_name},
-        query_params={'wait': 'true'},
-        body={
-            "payload": {"test_payload": "keyword"},
-            "points": [6]
-        }
+        path_params={"collection_name": collection_name},
+        query_params={"wait": "true"},
+        body={"payload": {"test_payload": "keyword"}, "points": [6]},
```

Comment:
I'm not sure how I feel about collapsing the `body` object everywhere. I think that what we had was much easier to read (and edit). What do you think?

Reply:
It's what my auto formatter (`black`) did, so no conscious decision. I agree that having it over multiple lines is more readable and creates better diffs when adding or removing fields (trailing commas help with that as well), so I'll change it back